AutoTenez.py

```diff
@@ -137,13 +139,54 @@ def make_reservation(headers, cookies, date_tomorrow, md5slotkey, your_external_
     print("ERROR! Enter your email address and password")
     sys.exit(0)
 
+# Parse input arguments
+parser = argparse.ArgumentParser(description='Reserve tennis court for tomorrow.')
+parser.add_argument('-c',  '--courts',               nargs='+', help='Specify courts ("Baan X", where X is the court number). Default setting is all courts.')
+parser.add_argument('-d',  '--date',                            help='Specify the date to make the reservation (yyyy-mm-dd)')
+parser.add_argument('-t2', '--time_second_choice',   nargs='+', help='Time you would like to reserve for the second option (hh:mm).')
+parser.add_argument('-c2', '--courts_second_choice', nargs='+', help='Specify courts for the second option.')
+
+required_arguments = parser.add_argument_group('Required arguments')
+required_arguments.add_argument('-t', '--time',    nargs='+', help="Time you would to you reserve. One or two consecutive times are allowed (hh:mm).", required=True)
+required_arguments.add_argument('-f', '--friends', nargs='+', help="With whom you would like to play. 1-3 additional players allowed.", required=True)
+
+args = parser.parse_args()
+
+player2_external_reference = args.friends[0]
+if len(args.friends) > 1:
+    player3_external_reference = args.friends[1]
+if len(args.friends) > 2:
+    player4_external_reference = args.friends[2]
+
+reservation_date = args.date
+
+first_choice_first_hour = args.time[0]
+if len(args.time) > 1:
+    first_choice_second_hour = args.time[1]
+
+if args.courts:
+    first_choice_courts = args.courts
+else:
+    first_choice_courts = []
```

Comment:
```suggestion
first_choice_courts = []
if args.courts:
    first_choice_courts = args.courts
```
Which is more consistent with the rest of the code.